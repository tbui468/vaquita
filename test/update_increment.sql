create database sol;
open sol;

create table planets (id int8 key, name text, moons int8);
insert into planets (id, name, moons) values (1, "Neptune", 20), (2, "Mars", 10), (3, "Venus", 10);
update planets set moons = moons + 100;
select * from planets;
drop table planets;

close sol;
drop database sol;
exit;
