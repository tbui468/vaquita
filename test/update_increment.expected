created database sol
opened database sol
created table planets
inserted 3 record(s) into planets
3 row(s) updated
id, name, moons
1, Neptune, 120
2, Mars, 110
3, Venus, 110
dropped table planets
closed database sol
dropped database sol
